parser_benchmark/src/bin/fuzz.rs

```diff
@@ -0,0 +1,135 @@
+use std::env::temp_dir;
```

Comment:
I guess I'd prefer to use cargo-fuzz for this, at the top level of the repo. Any reason you didn't go that way?